tests/smoke/support/page/files/allFiles.ts

```diff
@@ -110,12 +124,24 @@ export class AllFilesPage {
       await cta.files.navigateToFolder({ page: page, path: folderPaths.join('/') })
     }
 
-    await cta.files.sidebar.open({ page: page, resource: folderName })
-    await cta.files.sidebar.openPanel({ page: page, name: 'sharing' })
+    if (mainMenu) {
+      const element = await page.waitForSelector(
+        `//*[@data-test-resource-name="${folderName}"]/ancestor::tr//button[contains(@class, "files-quick-action-collaborators")]`
+      )
```

Comment:
feel free to add test selectors in web, xpath is a handy way of selecting an element but also error prone. That's not a show stopper... just wanted to mention it